tools/install.py

```diff
@@ -118,10 +118,13 @@ def files(action):
     if is_windows:
       output_file += '.dll'
     else:
-      # GYP will output to lib.target, this is hardcoded in its source,
-      # see the _InstallableTargetInstallPath function.
-      output_prefix += 'lib.target/'
-      output_file = 'lib' + output_file + '.so.' + get_version()
+      if sys.platform== 'darwin':
```

Comment:
Style: space before `==`.  You can fold the `if` into the preceding `else`, saves a level of indent.

(In case you're not a pythonista, the keyword is `elif`.)